app/controllers/AlertController.java

```diff
@@ -61,6 +72,89 @@ public AlertController(final Configuration configuration, final AlertRepository
     }
 
     /**
+     * Adds an alert in the alert repository.
+     *
+     * @return Created if the alert was created successfully, an error code otherwise.
+     */
+    public Result add() {
+        Alert alert;
```

Comment:
I believe with java 8 you can declare this final since the try-catch either sets it or returns. You may have to include line 82 inline on line 83.